srml/contracts/src/exec.rs

```diff
@@ -753,7 +753,7 @@ where
 	}
 
 	fn random(&self, subject: &[u8]) -> SeedOf<T> {
-		randomness_collective_flip::Module::<T>::random(subject)
+		<T::Randomness as Randomness<T::Hash>>::random(subject)
```

Comment:
```suggestion
		T::Randomness::random(subject)
```

I don't think that we need this cast?